leakpro/attacks/utils/boosting.py

```diff
@@ -136,7 +136,7 @@ def _memorization_score(self:Self) -> None:
             #       filtering of IN- vs. OUT-samples.
             target_logits = np.swapaxes(logits_function([self.target_model], self.handler, self.audit_data_indices,\
                                         self.batch_size), 0, 1).squeeze()
-            target_logits = self.softmax_logits(target_logits)
+            target_logits = softmax_logits(target_logits)
```

Comment:
Since you seems to be be setting the temp for all softmax uses in RMIA, I suggest to change the default value for temp to 1 or set it to one here

Reply:
done.